integration/cli/build.nonconflict.test.ts

```diff
@@ -0,0 +1,187 @@
+          expect(watcherImage).toBe(customImage);
+        }
+
+        const zarfYaml = await resource.oneFromFile(`${outputDir}/zarf.yaml`);
+        {
+          const componentImage = zarfYaml.components.at(0).images.at(0);
+          expect(componentImage).toBe(customImage);
+        }
+
+        const valuesYaml = await resource.oneFromFile(`${outputDir}/${uuid}-chart/values.yaml`);
+        {
+          const admissionImage = valuesYaml.admission.image;
+          expect(admissionImage).toBe(customImage);
+
+          const watcherImage = valuesYaml.watcher.image;
+          expect(watcherImage).toBe(customImage);
+        }
+      });
+
+      it("--timeout, works", async () => {
+        const moduleYaml = await resource.manyFromFile(`${outputDir}/pepr-module-${uuid}.yaml`);
+        {
+          const mwc = resource.select(
+            moduleYaml,
+            kind.MutatingWebhookConfiguration,
+            `pepr-${uuid}`,
+          );
+          const webhook = mwc.webhooks!.filter(f => f.name === `pepr-${uuid}.pepr.dev`).at(0)!;
+          expect(webhook.timeoutSeconds).toBe(timeout);
+        }
+        {
+          const mwc = resource.select(
+            moduleYaml,
+            kind.ValidatingWebhookConfiguration,
+            `pepr-${uuid}`,
+          );
+          const webhook = mwc.webhooks!.filter(f => f.name === `pepr-${uuid}.pepr.dev`).at(0)!;
+          expect(webhook.timeoutSeconds).toBe(timeout);
+        }
+
+        const valuesYaml = await resource.oneFromFile(`${outputDir}/${uuid}-chart/values.yaml`);
+        expect(valuesYaml.admission.webhookTimeout).toBe(timeout);
+      });
+
+      it("--withPullSecret, works", async () => {
+        const getDepImgPull = (deploy: kind.Deployment): string[] => {
+          return deploy!.spec!.template!.spec!.imagePullSecrets!.map(m => m.name!);
+        };
+
+        const moduleYaml = await resource.manyFromFile(`${outputDir}/pepr-module-${uuid}.yaml`);
+        const admission = resource.select(moduleYaml, kind.Deployment, `pepr-${uuid}`);
+        const admissionSecrets = getDepImgPull(admission);
+        expect(admissionSecrets).toEqual([withPullSecret]);
+
+        const watcher = resource.select(moduleYaml, kind.Deployment, `pepr-${uuid}-watcher`);
+        const watcherSecrets = getDepImgPull(watcher);
+        expect(watcherSecrets).toEqual([withPullSecret]);
+
+        // TODO: team talk
+        // Image pull secrets don't seem to map into the Helm chart anywhere..? Should they?
+        // TODO: end
+      });
+
+      it("--zarf, works", async () => {
+        const chart = {
+          name: "module",
+          namespace: "pepr-system",
+          version: "0.0.1",
+          localPath: `${uuid}-chart`,
+        };
+
+        const zarfYaml = await resource.oneFromFile(`${outputDir}/zarf.yaml`);
+        const component = zarfYaml.components
+          .filter((f: { name: string }) => f.name === "module")
```

Comment:
```suggestion
          .filter((file: { name: string }) => file.name === "module")
```

Does `f` represent a `file` here?